### PR TITLE
Abstracts

### PR DESCRIPTION
**Pull request by SolarDrew**
I'm missing a few abstracts and, unfortunately, a few titles for this page. If anyone has time to track those down, that would be great.

### CODE REVIEW

File: pages/2018/abstracts.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+For modern CPUs, such a tuning involves proper utilization of the cache hierarchy, vectorized code targeting the Single Instruction Multiple Data (SIMD) capable wide vector registers as well as {\em many-core} parallelization.
+Here I present Corrfunc --  a suite of OpenMP-parallelized clustering codes that exploit current CPU micro-architecture with custom Advanced Vector Extensions (AVX) and Streaming SIMD Extensions (SSE) intrinsics.
+Corrfunc can compute a variety of correlation functions for source positions in either a Cartesian geometry (i.e., generated from cosmological simulations) or for positions on the sky.
+The algorithm within Corrfunc can be easily adapted to a variety of different measurements and has already been implemented for nearest neighbour searches, group finding in galaxy surveys, weak lensing measurements etc.
+By design, Corrfunc is highly optimized and can compute \wprp for \mathcal{O}(1 million) galaxies in ~ 6 seconds on a post-2011 CPU, which is at least a factor of few faster than existing public correlation function routines.
```
Comment (SolarDrew) [inline pages/2018/abstracts.rst]: If anybody knows how to latex in rst I'd appreciate some pointers here.

File: pages/2018/abstracts.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
+.. title: Talk titles and abstracts
+
+Something about PyAstro - Karl Gordon
```
Comment (mwcraig) [inline pages/2018/abstracts.rst]: Titile is: How Python in Astronomy has Impacted My Professional Life

File: pages/2018/abstracts.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+In this talk I will give an overview of the software under development to enable users to perform scientific analysis on the data collected by the DKIST.
+This software is being developed in Python using SunPy and Astropy to provide the core tools.
+The DKIST data centre is expected to generate approximately 3 Pb of calibrated science quality data a year once telescope operations are at full capacity.
+The data centre will provide no pre-processing of the data before download, which is an extra challenge to the user software.
+The tools that will be written for these data will include, search and retrieval from the data centre, cataloguing and searching data local to the user and then reading of this data and metadata in a way which is compatible with existing tooling.
+This talk will describe these stages and the new and existing tools used in implementing them, as well as the challenges faced while doing so.
+
+Collaborations in the Extreme: the rise of open code development in the scientific community - Kelle Cruz
+#########################################################################################################
```
Comment (mwcraig) [inline pages/2018/abstracts.rst:96]: See this page for abstract: https://www.simonsfoundation.org/event/collaborations-in-the-extreme-the-rise-of-open-code-development-in-the-scientific-community/

**Review by mwcraig — COMMENTED** [review]
Got at least a couple of the details in. Have asked Karl for an abstract.

**Comment by mwcraig** [comment]
LGTM

**Comment by SolarDrew** [comment]
Thanks @mwcraig . Will merge once the checks pass.

**Pull request merged**